add usage example to reduce_min

mrinalTheCoder opened the pull request:
I'm participating in GCI '19 (handle: jimmy_page). Task: [Advanced] Add a usage example to TensorFlow 2.0 API documentation.

I have added a usage example to tf.math.reduce_min.

mrinalTheCoder committed:
add usage example to reduce_min

gbaned commented:
@mrinalTheCoder thank you, it is failing doctest can you please check here for [logs](https://source.cloud.google.com/results/invocations/f9a41ff4-e8d7-46da-b08f-29989b01aee4/targets/%2F%2Ftensorflow%2Ftools%2Fdocs:tf_doctest/tests).

Please run the doctest locally as mentioned here in the [contributor guidelines](https://www.tensorflow.org/community/contribute/docs_ref).

mrinalTheCoder committed:
fixed failing doctest

mrinalTheCoder commented:
I fixed the failing doctest in the latest commit. It was because my machine was running tf 2.0 whereas the doctest was done on tf 2.1
I have fixed the output of the doctest in the file, please review.
